### ISSUE TITLE
Clojure no way to re-publish a translation

### ISSUE THREAD

**Issue by g964** [Codewars/codewars.com#1270]
I am trying to republish:
https://www.codewars.com/kata/554f76dca89983cc400000bb/edit/clojure
after adding random tests.
I validate without problem but when I re-publish I get:
"Final Solution is invalid. (clojure) The solution should pass the provided test cases"
... even if I try to re-publish the *old* version (the one without random tests) which has been done long ago and passed by Codewarriors.

**Comment by Voileexperiments** [comment]
I think the default workaround on that is to fork the translation.

It says nothing about *why* editing translations fails at random scenarios (e.g you can't edit an already approved translation), but it should still work.

**Comment by g964** [comment]
I forked it but unable to publish because of merging conflicts. Somebody had modified the description (I have not been warned of that). I suppressed the modifications in the description and this time I got "Published with failed tests". In fact the fork has not been published...

What if I suppress the Clojure translation and publish anew with random tests added?

Too often translations at CW is a real mess: in this case + two other katas ( in one with Elixir where the text is invisible and when you save the edit panel becomes entirely white; in another where I always get a 500 error when trying to add a R translation!)
